packages/react/package.json

```diff
@@ -21,6 +21,17 @@
     "dist",
     "web"
   ],
+  "keywords": [
+    "lens-api",
+    "lens-protocol-sdk",
+    "lens-protocol",
+    "lens",
+    "react",
+    "sdk",
+    "social",
+    "web3-social",
+    "web3"
+  ],
```

Comment:
Maybe we should leave this one out. it might become internal as soon as the RN one is stable.